tests/behaviour/expectations/behaviour.ts

```diff
@@ -6,6 +6,43 @@ export const expectations = flatten(
     new Dir('behaviour', [
       new Dir('contracts', [
         new Dir('abiEncode', [
+          File.Simple('abiDecodeValue', [
+            Expect.Simple('decodeAsInt24', getByte32Array(148), ['148']),
+            Expect.Simple('decodeAsUint256', getByte32Array(0), ['0', '0'], 'Zero uint256'),
+            Expect.Simple('decodeAsUint256', getByte32Array(148), ['148', '0'], 'Small uint256'),
+            Expect.Simple(
+              'decodeAsUint256',
+              getByte32Array(BigInt(2) ** BigInt(128) - BigInt(1)),
+              [`${BigInt(2) ** BigInt(128) - BigInt(1)}`, '0'],
```

Comment:
tip you can write a BigInt literal by putting a `n` suffix on a number
example: BigInt(2) -> 2n